src/providers/database.ts

```diff
@@ -22,19 +22,19 @@
 
 import { apps } from '../apps';
 import {
-  Change,
   CloudFunction,
   Event,
   EventContext,
   makeCloudFunction,
 } from '../cloud-functions';
+import { Change } from '../common/core';
 import { DataSnapshot } from '../common/providers/database';
 import { firebaseConfig } from '../config';
 import { DeploymentOptions } from '../function-configuration';
 import { normalizePath } from '../utilities/path';
 import { applyChange } from '../utils';
 
-export { DataSnapshot };
+export { DataSnapshot, Change };
```

Comment:
I don't think we actually exported Change from the RDTB namespace in the past, did we? IIRC it was just exported in `index.ts`. We should put it in `v1/index.ts` and `v2/index.ts` as well as `v2/core.ts`